source/libs/page-detect.ts

```diff
@@ -414,3 +414,8 @@ export const _isActionPage = [
 	'https://github.com/marketplace/actions/github-action-for-assignee-to-reviewer',
 	'https://github.com/marketplace/actions/hugo-actions'
 ];
+
+export const _hasNotificationBar = skip;
+export const hasNotificationBar = (): boolean =>
+	location.search.startsWith('?notification_referrer_id=') ||
+	JSON.parse(sessionStorage.notification_shelf ?? '{}').pathname === location.pathname;
```

Comment:
This is too specific to be part of `page-detect`, can you move it to the feature’s file?

Reply:
Done.